web-portal/store/index.js

```diff
@@ -1,7 +1,7 @@
 import { ApiService } from '../services/ApiService'
 import { getEmptyCalendarEvent } from '../services/ResourceTemplateService'
 
-const CURRENT_EVENTS_VERIFIED_PATH = '/current-events/verified'
+const CURRENT_EVENTS_VERIFIED_PATH = '/v1/current-events/verified'
```

Comment:
should the /v1/ part not just be set in the api url in the evn?

Reply:
Yep. I hadn't updated my `web-portal/.env` file.